HW1/Task_1.py

```diff
@@ -0,0 +1,30 @@
+# 2.Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+# Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны
+# с суммой двух других. Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника
+# с такими сторонами не существует. Отдельно сообщить
+# является ли треугольник разносторонним, равнобедренным или равносторонним.
+def check_delta (a, b, c):
+    if a == b and a == c:
+        return 'раносторонний'
+    elif a == b or a == c or b == c:
+        return 'ранобедренный'
+    else:
+        return 'разносторонний'
+
+print('введите стороны треугольника')
+delta = (int(input()), int(input()), int(input()))
```

Comment:
Nnen скобки не нужны. 
```suggestion
delta = int(input()), int(input()), int(input())
```